### PR TITLE
Add .NET 9 nightly NuGet feed and remove .NET 7

### PR DESCRIPTION
**Pull request by ctyar**
Just added a new section for .NET 9 and removed .NET 7 to keep docs updated with the latest versions.

### CODE REVIEW

File: docs/DailyBuilds.md [hunk 1]
```diff
@@ -7,27 +7,27 @@ If you want to download the latest daily build and use it in a project, then you
...
-## .NET 7
+## .NET 8
 
   ```xml
   <?xml version="1.0" encoding="utf-8"?>
   <configuration>
       <packageSources>
           <clear />
-          <add key="aspnetcore" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json" />
+          <add key="aspnetcore" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
```
Comment (captainsafia) [inline docs/DailyBuilds.md:17]:
```suggestion
          <add key="dotnet8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
```

File: docs/DailyBuilds.md [hunk 2]
```diff
@@ -7,27 +7,27 @@ If you want to download the latest daily build and use it in a project, then you
...
+
+## .NET 9
 
   ```xml
   <?xml version="1.0" encoding="utf-8"?>
   <configuration>
       <packageSources>
           <clear />
-          <add key="aspnetcore" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
+          <add key="aspnetcore" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet9/nuget/v3/index.json" />
```
Comment (captainsafia) [inline docs/DailyBuilds.md:30]:
```suggestion
          <add key="dotnet9" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet9/nuget/v3/index.json" />
```

**Review by captainsafia — APPROVED** [review]
LGTM! Proposing a small tweak to also make it clear that these feeds are not ASP.NET Core-specific.

**Comment by ctyar** [comment]
Updated the keys

**Comment by mkArtakMSFT** [comment]
Thanks for reviewing this, @captainsafia.
Leaving this for you to merge once the CI checks pass and you're still happen with it.

**Comment by captainsafia** [comment]
@dotnet/aspnet-build Can I get help merging this?

**Comment by mgravell** [comment]
I'll look this morning @captainsafia

**Pull request merged**